packages/react-data-grid/src/Cell.tsx

```diff
@@ -1,22 +1,16 @@
 import React, { memo } from 'react';
 import classNames from 'classnames';
 
-import CellActions from './Cell/CellActions';
-import CellContent from './Cell/CellContent';
-import CellExpand from './Cell/CellExpander';
 import { CellRendererProps, ColumnEventInfo } from './common/types';
 import { isFrozen } from './utils/columnUtils';
 
 export interface CellProps<R> extends CellRendererProps<R> {
-  children?: React.ReactNode;
   // TODO: Check if these props are required or not. These are most likely set by custom cell renderer
+  children?: React.ReactNode;
```

Comment:
I believe we don't have the usage in canary code base internally from beginning(?) or at least for a long time. To have the `children` in `Cell`, we need to render it in `Row.tsx` like `<Cell><Something /></Cell>` in `getCells` function, However, [this line](https://github.com/adazzle/react-data-grid/blob/canary/packages/react-data-grid/src/Row.tsx#L59) directly renders the `Cell` (`Renderer`) and passes no children inside.

One other possibility is that since we directly export `Cell` component, potentially people could do `<Cell><TheirCode /></Cell>` in their code base. I believe it's no harm to keep it for now and we might need to decide to keep or kill it with `column.formatter` together, I believe we don't need to rush in this PR.